### PR TITLE
Creation page calendar, card calendar, add route

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by Hamza-Luciano — APPROVED** [review]

File: frontend/src/components/CardCalendar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import PropTypes from "prop-types";
+import SCardCalendar from "@components/CardCalendar/style";
+import Title from "@components/Archi/Title";
+import domicile from "../../assets/Pictures/cards/calendar/picto-maison.png";
```
Comment (LuigiMenez) [inline frontend/src/components/CardCalendar/index.jsx:4]: Use import absolute

File: frontend/src/components/CardCalendar/style.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    color: var(--white);
+    h3 {
+      margin: 0 10px 0 10px;
+    }
+  }
+  .link {
+    background-color: var(--yellow);
+    margin: 1vw;
+    color: var(--white);
+    /* text-shadow: 2px 2px 5px black;
```
Comment (LuigiMenez) [inline frontend/src/components/CardCalendar/style.jsx:83]: No Dead code

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Use import absolute & dead code

File: frontend/src/components/CardCalendar/index.jsx (hunk 1 above)
Comment (Gonaline) [reply frontend/src/components/CardCalendar/index.jsx]: ok

File: frontend/src/components/CardCalendar/style.jsx (hunk 2 above)
Comment (Gonaline) [reply frontend/src/components/CardCalendar/style.jsx]: ok

**Review by FrancoisSalmon — APPROVED** [review]
no more dead code and absolute import ok

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix conflict

**Review by FrancoisSalmon — APPROVED** [review]
conflicts resolved

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**